src/test/java/net/yageek/lambert/LambertTest.java

```diff
@@ -45,15 +45,19 @@ public void Lambert93BugTest() {
         LambertPoint pt = Lambert.convertToWGS84Deg(668832.5384, 6950138.7285, LambertZone.Lambert93);
         assertEquals(2.56865, pt.getX().doubleValue(), 0.0001);
         assertEquals(49.64961, pt.getY().doubleValue(), 0.0001);
+
     }
 
     @Test
     public void LambertIIExtendedToWgs84Test() {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

Reply:
Hello
I left the test method names exactly like they were in the initial repo.
Yes, this is bad practice, but fixing this was not part of my reasons for this pull request.
Thank you.

Reply:
noticed